core/modules/parsers/wikiparser/rules/filteredtranscludeinline.js

```diff
@@ -7,62 +7,44 @@ Wiki text rule for inline filtered transclusion. For example:
 
 ```
 {{{ [tag[docs]] }}}
-{{{ [tag[docs]] |tooltip}}}
 {{{ [tag[docs]] ||TemplateTitle}}}
-{{{ [tag[docs]] |tooltip||TemplateTitle}}}
-{{{ [tag[docs]] }}width:40;height:50;}.class.class
+((( [tag[docs]] )))
+((( [tag[docs]] ||TemplateTitle)))
```

Comment:
I am not happy about this one. There are 2 wikitext syntax variants now, that do the exact same thing.